proposals/README.md

```diff
@@ -0,0 +1,6 @@
+# Azure Kinect Sensor SDK Proposals
+
+New features proposals are living documents describing the current thinking about a given feature. Proposals are the way for Azure Kinect team  to communicate to the public upcoming new feature work and get the feedback.
```

Comment:
Can we add a couple sections here? 
* One about the stages or the proposal process from initial idea to completed feature.
* One about where users should direct feedback (the proposal up until the feature is checked in)

Reply:
added those changes